hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -486,10 +485,23 @@ private void cacheConfigXmlGenerator(StringBuilder xml, Config config) {
         }
     }
 
+    private void checkAndFillCacheWriterFactoryConfigXml(StringBuilder xml, String cacheWriter) {
+        if (!StringUtil.isNullOrEmpty(cacheWriter)) {
+            xml.append("<cache-writer-factory class-name=\"").append(cacheWriter).append("\"/>");
+        }
+    }
+
+    private void checkAndFillCacheLoaderFactoryConfigXml(StringBuilder xml, String cacheLoader) {
+        if (!StringUtil.isNullOrEmpty(cacheLoader)) {
+            xml.append("<cache-loader-factory class-name=\"").append(cacheLoader).append("\"/>");
+        }
+
```

Comment:
I think these two methods can be merged into a single one accepting xml tag name as an additional param.